### PR TITLE
Handles decimal powers

### PR DESCRIPTION
**Pull request by ghost**
## Notes
1. This power function does not handle complex numbers. An answer containing a complex number will return an error.
2. Changes made to the power function by @steveiaco has not yet been implemented.

### CODE REVIEW

**Comment by steveiaco** [comment]
Looks good to me! 

For 2, the changes I made to the function on my branch will automerge with your (or we'll get a merge conflict that we'll fix). So it's all good!

**Comment by steveiaco** [comment]
Also, please move the test function and the call to it to a separate file, as if we import this file it'll autorun that function no?

File: Functions/Power_Function.py [hunk 1]
```diff
@@ -1,29 +1,82 @@
 #Goal: Calculate the specified power of a specified base
-#Method: Exponentiation by squaring 
+#Handling: Does not compute complex numbers 
+
+from math import floor
+from math import sqrt
```
Comment (seanheinrichs) [inline Functions/Power_Function.py:5]: these are acceptable imports?

**Review by seanheinrichs — APPROVED** [review]
lgtm

**Pull request merged**